ui/src/components/giving/GivingCard.js

```diff
@@ -59,9 +59,9 @@ const RenderSwitch = (param) => {
             <Text fontWeight="bold">Bank Code: 009</Text>
             <Text fontWeight="bold">Branch Code: 845</Text>
             <Text fontWeight="bold">
-              Account Name: Harvest Mission Community Church (Hong Kong)
+              Account Name: Harvest Mission Community Church (Hong Kong) Limited
             </Text>
-            <Text fontWeight="bold">Limited Account Number: 13012090</Text>
+            <Text fontWeight="bold"> Account Number: 013012090</Text>
```

Comment:
how come we added a 0 in front? is this intentional?

Reply:
yep, turns out the correct bank acc should have the 0 in the front